test/features/data_qld_theme.feature

```diff
@@ -0,0 +1,39 @@
+@data-qld-theme
+Feature: Data QLD Theme
+
+       Scenario: Lato font is implemented on homepage
+              When I go to homepage
+              Then I should see an element with xpath "//link[contains(@href,'https://fonts.googleapis.com/css?family=Lato')]"
+
+       Scenario: Organisation is in fact spelled Organisation (as opposed to Organization) 
+              When I go to organisation page
+              Then I should not see "Organization"
+
+       Scenario: Explore button does not exist on dataset detail page
+              When I go to dataset page
+              And I click the link with text that contains "A Wonderful Story"
+              Then I should not see "Explore"
+
+       Scenario: Explore button does not exist on dataset detail page
+              When I go to organisation page
+              Then I should see "Organisations are Queensland Government departments, other agencies or legislative entities responsible for publishing open data on this portal."
+
+      Scenario: Register user password must be 10 characters or longer
+              When I go to register page
+              And I fill in "name" with "name"
```

Comment:
Here's some code we use for another application which handles most/all types of data entry, this reduces the number of And lines and will fill in the order that they are given.

Cucumber table data may be better for this type of fill in.
```
And I enter the following:
  | key | type | value	| name-hint |
  | Username| text| name | name |
  | Full Name | text | fullname | fullname|
  | Email| email| email@test.com | email |
  | Password | password | pass| pasword1 |
  | Confirm | password | pass | password2 |
```
or without name-hint if no label will duplicate
```
And I enter the following:
  | Username| text| name | name |
  | Full Name | text | fullname | 
  | Email| email| email@test.com | 
  | Password | password | pass| 
  | Confirm | password | pass | 
```

in our java code for cucumber we use this please use this as an example to convert to python version
```
@And("^I enter the following:$")
    public void enterTheFollowing(DataTable dataTable) throws Throwable {
        //always have first line of headers
        //|key|type| value|
        int key = -1;
        int type = -1;
        int value = -1;
        int nameHint = -1;
        List<String> headerRow = dataTable.getGherkinRows().get(0).getCells();
        for (int i = 0; i < headerRow.size(); i++) {
            if ("key".equalsIgnoreCase(headerRow.get(i))) {
                key = i;
            } else if ("type".equalsIgnoreCase(headerRow.get(i))) {
                type = i;
            } else if ("value".equalsIgnoreCase(headerRow.get(i))) {
                value = i;
            } else if ("name-hint".equalsIgnoreCase(headerRow.get(i))) {
                nameHint = i;
            }
        }

        boolean headerSeen = false;
        for (DataTableRow row : dataTable.getGherkinRows()) {
            if (headerSeen) {
                List<String> data = row.getCells();
                if (nameHint != -1) {
                    fillInForm(data.get(key), data.get(type), data.get(value), data.get(nameHint));
                } else {
                    fillInForm(data.get(key), data.get(type), data.get(value), null);
                }
            } else {
                headerSeen = true;
            }
        }
    }

public void fillInForm(String label, String type, String value, String nameHint) {
        String xpath;
        if ("text".equalsIgnoreCase(type) || "tel".equalsIgnoreCase(type) || "email".equalsIgnoreCase(type) || "date".equalsIgnoreCase(type)) {
            if (StringUtils.isNotBlank(label)) {
                if (StringUtils.isNotBlank(nameHint)) {
                    xpath = "//label/span[contains(text(), \"" + label + "\")]/../../input[@type='" + type + "' and @name='" + nameHint + "']";
                } else {
                    xpath = "//label/span[contains(text(), \"" + label + "\")]/../../input[@type='" + type + "']";
                }
            } else {
                if (StringUtils.isNotBlank(nameHint)) {
                    xpath = "//input[@type='" + type + "' and @name='" + nameHint + "']";
                } else {
                    Assertions.fail("nameHint must be set went label is blank, value: {}, type {}", value, type);
                    xpath = "";
                }
            }
            ic.waitForElementorElementsToBeDisplayedXP(xpath);
            WebElement element = ic.findElementByXpath(xpath);
            if (element == null) {
                Assertions.fail("could not find label, type, did find label: %s, %s, %s", label, type, value);
            }
            if (!StringUtils.equalsIgnoreCase("date", type)) {
                element.clear();
            }
            element.sendKeys(value);
            if ("Residential address".equalsIgnoreCase(label)) { //could be via type instead
                ic.waitForElementorElementsToBeDisplayedXP("//div[@class='pac-item']");
                element.sendKeys(Keys.DOWN);
                try {
                    TimeUnit.MICROSECONDS.sleep(100L);
                } catch (InterruptedException e) {
                    e.printStackTrace();
                }
                element.sendKeys(Keys.TAB);
                try {
                    TimeUnit.SECONDS.sleep(1L);
                } catch (InterruptedException e) {
                    e.printStackTrace();
                }
            } else {
                element.sendKeys(Keys.TAB);
                try {
                    TimeUnit.MICROSECONDS.sleep(100L);
                } catch (InterruptedException e) {
                    e.printStackTrace();
                }
            }
        } else if ("radio".equalsIgnoreCase(type)) {
            if (StringUtils.isNotBlank(nameHint)) {
                xpath = "//span[contains(text(),\"" + label + "\")]/../..//input[@type=\"radio\" and @name=\"" + nameHint + "\"]/../label[contains(., \"" + value + "\")]";
            } else {
                xpath = "//span[contains(text(),\"" + label + "\")]/../..//input[@type=\"radio\"]/../label[contains(., \"" + value + "\")]";
            }
            ic.waitForElementorElementsToBeDisplayedXP(xpath);
            WebElement element = ic.findElementByXpath(xpath);
            if (element == null) {
                Assertions.fail("could not find label, type, did find label: %s, %s, %s", label, type, value);
            }
            try {
                TimeUnit.MICROSECONDS.sleep(300L);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
            element.click();
            try {
                TimeUnit.MICROSECONDS.sleep(300L);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
        } else if ("select".equalsIgnoreCase(type)) {
            xpath = "//label/span[contains(text(), \"" + label + "\")]/../../select/option[contains(text(),\"" + value + "\")]";
            WebElement element = ic.findElementByXpath(xpath);
            if (element == null) {
                Assertions.fail("could not find label, type, did find label: %s, %s, %s", label, type, value);
            }
            element.click();
        } else if ("checkbox-single".equalsIgnoreCase(type)) {
            String nameHintXpath = "";
            if (StringUtils.isNotBlank(nameHint)) {
                nameHintXpath = " and @name=\"" + nameHint + "\"";
            }
            xpath = "//label[contains(text(), \"" + label + "\")]/../input[@type='checkbox'" + nameHintXpath + "]";
            WebElement element = ic.findElementByXpath(xpath);
            if (element == null) {
                xpath = "//label[b[contains(text(), \"" + label + "\")]]/../input[@type='checkbox'" + nameHintXpath + "]";
                element = ic.findElementByXpath(xpath);
            }
            if (element == null) {
                Assertions.fail("could not find label, type, did find label: %s, %s, %s", label, type, value);
            }
            element.click();
        } else if ("scroll".equalsIgnoreCase(type)) {
            ((JavascriptExecutor) secWebDriverHolder.getWebDriver())
                    .executeScript("document.getElementById('" + label + "').scrollBy(0,10000);");
        }
    }
```